### DISCUSSION TITLE
Layout route is malformed in express

### DISCUSSION THREAD

**Discussion by aristizabal95** [harttle/liquidjs#292] [Working with Express]
I'm trying to use layouts in my express app but the app fails with this error
```
There was a problem RenderError: ENOENT: Failed to lookup "default" in "./views,.",
```
I'm using the `express-generator` `app.js` template, and modified it to require Liquidjs like this:
```javascript
const { Liquid } = require('liquidjs');
const engine = new Liquid();

var indexRouter = require('./routes/index');
var usersRouter = require('./routes/users');

var app = express();

// view engine setup
app.engine('liquid', engine.express());
app.set('views', './views');
app.set('view engine', 'liquid');
```
As indicated on the guides you provide. 
The `index.liquid` file only loads the layout and adds "Hello World".
```
{% layout "default" %}
{% block content %}Hello World.{% endblock %}
```
Once I remove the layout line everything works.
It seems that the layout route has an ",." appended to the end.

**Comment by harttle** [comment]
Put a '.liquid' in `layout` tag:

```
{% layout "default.liquid" %}
```

Or set it as a default when creating Liquid instance:

```javascript
const engine = new Liquid({ extname: '.liquid' });
```

I'll try to make it clear in [tutorials](https://liquidjs.com/tutorials/partials-and-layouts.html).

**Comment by harttle** [comment]
The `"./views,."` string is the result of `["./views", "."].toString()`, of which the first path is set by express views, the second path is set by LiquidJS as a default path.
